### PR TITLE
ft: ZENKO-584 record failed crr metrics

### PR DESCRIPTION
**Pull request by philipyoo**
When replication status is reported for a given site on the source object,
if "COMPLETED", report completion metrics, and if "FAILED", report failed
metrics.

Known issue reported on jira tag ZENKO-621

Relies on: https://github.com/scality/Arsenal/pull/524

### CODE REVIEW

File: docs/metrics.md [hunk 1]
```diff
@@ -75,26 +77,46 @@ to **15 minutes**.
...
-    of MB transferred (size) in the last 900 seconds",
+    of bytes transferred (size) in the last 900 seconds",
     "results":{
         "count":31,
         "size":"47.04"
     }
 }
 ```
 
+### `/_/metrics/crr/<site-name>/failures`
```
Comment (jonathan-gramain) [inline docs/metrics.md]: Can you change this and others to `<location-name>` as well?

File: tests/unit/api/BackbeatAPI.spec.js [hunk 2]
```diff
@@ -49,6 +50,8 @@ describe('BackbeatAPI', () => {
             { url: '/_/metrics/crr/test', method: 'GET' },
             { url: '/_/metrics/crr/all/backlo', method: 'GET' },
             { url: '/_/metrics/crr/all/completionss', method: 'GET' },
+            { url: '/_/metrics/crr/all/failures', method: 'POST' },
+            { url: '/_/metrics/crr/all/fail', method: 'GET' },
```
Comment (jonathan-gramain) [inline tests/unit/api/BackbeatAPI.spec.js:54]: Is this line needed?

**Review by jonathan-gramain — APPROVED** [review]

File: tests/unit/api/BackbeatAPI.spec.js (hunk 2 above)
Comment (philipyoo) [reply tests/unit/api/BackbeatAPI.spec.js:54]: no it isn't required, just tests that the route here should fail to parse. Should I remove?

Comment (jonathan-gramain) [reply tests/unit/api/BackbeatAPI.spec.js:54]: no that's fine

**Review by alexanderchan-scality — APPROVED** [review]

**Review by rahulreddy — APPROVED** [review]

**Pull request merged**